docs/design/autoware-architecture/sensing/data-types/radar-data/radar-pointcloud-data.md

```diff
@@ -0,0 +1,75 @@
+# Radar pointcloud data pre-processing design
+
+## Overview
+
+### Pipeline
+
+This diagram describes the pre-process pipeline for radar pointcloud.
+
+![radar-pointcloud-sensing](image/radar-pointcloud-sensing.drawio.svg)
+
+### Interface
+
+- Input
+  - Radar data from device
+  - Twist information of ego vehicle motion
+- Output
+  - Dynamic radar pointcloud (`ros-perception/radar_msgs/msg/RadarScan.msg`)
+  - Noise filtered radar pointcloud (`sensor_msgs/msg/Pointcloud2.msg`)
+
+### Note
+
+- In the sensing layer, the radar pre-process packages filter noise through the `ros-perception/radar_msgs/msg/RadarScan.msg` message type with sensor coordinate.
+- For use of radar pointcloud data by LiDAR packages, we would like to propose a converter for creating `sensor_msgs/msg/Pointcloud2.msg` from `ros-perception/radar_msgs/msg/RadarScan.msg`.
+
+## Reference implementations
+
+### Data message for radars
+
+Autoware uses radar objects data type as [radar_msgs/msg/RadarScan.msg](https://github.com/ros-perception/radar_msgs/blob/ros2/msg/RadarScan.msg).
+In detail, please see [Data message for radars](reference-implementations/data-message.md).
+
+### Device driver for radars
+
+Autoware support `ros-perception/radar_msgs/msg/RadarScan.msg` and `autoware_auto_perception_msgs/msg/TrackedObjects.msg` for Radar drivers.
+
+In detail, please see [Device driver for radars](reference-implementations/device-driver.md).
+
+### Basic noise filter
+
+- [radar_threshold_filter](https://github.com/autowarefoundation/autoware.universe/tree/main/sensing/radar_threshold_filter)
+
+This package removes pointcloud noise with low amplitude, edge angle, too near pointcloud by threshold.
```

Comment:
[NITS]

Please add the reason for using this filter(e.g. the reason why too near pointcloud should be removed.)

Reply:
Fixed df7efc6